### PR TITLE
Add recipe for Cirrus CI

### PR DESCRIPTION
**Pull request by fkorotkov**
[Cirrus CI](https://github.com/marketplace/cirrus-ci) allows to use any Docker containers as execution environment so it was pretty easy to provide a recipe. 🎉Thanks for these awesome containers! 🙌

### CODE REVIEW

File: recipes/cirrus-ci/.cirrus.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+contianer:
```
Comment (mreichelt) [inline recipes/cirrus-ci/.cirrus.yml]: Should be:
```suggestion
container:
```

File: recipes/cirrus-ci/.cirrus.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  image: mreichelt/android:28
+  
+check_task:
```
Comment (mreichelt) [inline recipes/cirrus-ci/.cirrus.yml]: How about this:
```suggestion
build_task:
```
Would this more descriptive name work with Cirrus CI?

File: recipes/cirrus-ci/.cirrus.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+  check_script: ./gradlew check assemble
```
Comment (mreichelt) [inline recipes/cirrus-ci/.cirrus.yml]: Here as well:
```suggestion
  build_script: ./gradlew check assemble
```

**Review by mreichelt — CHANGES_REQUESTED** [review]
Nice - thanks for contributing! Didn't know about Cirrus CI yet. Can you check if Cirrus CI works with the suggested changes? I'll gladly merge your PR 🎉

**Comment by fkorotkov** [comment]
@mreichelt do you have a repo where you usually test?

**Comment by mreichelt** [comment]
Not yet ¯\\_(ツ)_/¯

**Review by mreichelt — APPROVED** [review]

**Pull request merged**